moduleroot/.github/workflows/publish.yaml.erb

```diff
@@ -0,0 +1,30 @@
+<% if configs['enabled'] -%>
+name: Build and publish to Puppet Forge
+
+on:
+ push:
+  tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get latest tag
+      run: echo ::set-env name=RELEASE_VERSION::$(echo ${GITHUB_REF:10})
+    - name: Clone repository
+      uses: actions/checkout@v1
```

Comment:
According to [github actions docs](https://github.com/actions/checkout#usage) the checkout action ref is automatically set to the reference or SHA for the event that triggered the action. I _think_ that means that its that tag, so the 'Get latest tag' step might be unnecessary

Reply:
I think you're correct. I just copied the example from that action's README, but the code actually grabs the release version from metadata, not the tag. https://github.com/barnumbirr/action-forge-publish/blob/eb321624ad542407801ff22cbf58426e4ed616e0/entrypoint.sh#L7

I have a few improvements to make to `node_encrypt`. Maybe I'll get some time to work on that this weekend and test publishing with this step removed.